analyzers/src/SonarAnalyzer.CSharp/Rules/InfiniteRecursion.RoslynCfg.cs

```diff
@@ -58,10 +58,12 @@ public void CheckForNoExitProperty(SyntaxNodeAnalysisContext c, PropertyDeclarat
 
             public void CheckForNoExitMethod(SyntaxNodeAnalysisContext c, CSharpSyntaxNode body, SyntaxToken identifier, IMethodSymbol symbol)
             {
-                var cfg = body.CreateCfg(c.SemanticModel);
-                var context = new RecursionContext<ControlFlowGraph>(cfg, symbol, identifier.GetLocation(), c, "method's recursion");
-                var walker = new RecursionSearcher(context);
-                walker.CheckPaths();
+                if (body.CreateCfg(c.SemanticModel) is { } cfg)
```

Comment:
Thanks for taking care of this. But if we get noisy - this commit could be reverted. Let's don't hide the errors